espnet2/enh/loss/criterions/time_domain.py

```diff
@@ -30,9 +31,12 @@ class CISDRLoss(TimeDomainLoss):
         loss: (Batch,)
     """
 
-    def __init__(self, filter_length=512):
+    def __init__(self, filter_length=512, clamp_db=None, use_cg_iter=None, load_diag=None):
```

Comment:
Could you add a docstring to explain the added arguments? It would help users to better understand how to use them.

Reply:
I have reverted to not use fast_bss_eval since @LiChenda is adding it in #4256 .